tag.php

```diff
@@ -20,7 +20,7 @@
 }
 
 $tag = get_queried_object();
-$redirect_id = $tag !== null && get_term_meta($tag->term_id, 'redirect_page', true);
+$redirect_id = isset($tag) ? get_term_meta($tag->term_id, 'redirect_page', true) : false;
```

Comment:
Do you think this can be safer for Sentry?

```suggestion
isset($tag, $tag->term_id) ? get_term_meta($tag->term_id, 'redirect_page', true) : false;
```

Reply:
Yes, good idea. Amended 👍🏻